docs/02_architecture_constraints.adoc

```diff
@@ -17,3 +17,53 @@ If needed you can subdivide them into
 technical constraints, organizational and political constraints and
 conventions (e.g. programming or versioning guidelines, documentation or naming conventions)
 ****
+We divide the constrains into two parts: technical and business constrains.
+
+=== Technical constraints
+
+[role="arc42help"]
+****
+.Contents
+The technical constraints for the development of the system.
+
+.Motivation
+Know the fixed technical constraints for the system.
+
+.Form
+Table with all the technical constraints and its corresponding description .
+****
+Technical constraints are those that are fixed technical constraints decisions. Most of them appear in the description of the system.
+
+[options="header",cols="1,2"]
+|===
+|Constraint|Description
+| _React_ | React defines itself JavaScript library for building user interfaces. The application has to be implemented usint React
+| _TypeScript_ | TypeScript is a strongly typed programming language that builds on JavaScript. 
+| _SOLID principles_|Solid is a specification that lets people store their data securely in decentralized data stores called Pods. When data is stored in someone's Pod, they control its access.
+| _Continuous integration system_|The application has to be accessible and deployed using a continuous integration system.
+| _Keep track of orders done_|The system has to allow users to visualize the orders done, so it has to keep track of them.
+|===
+
+=== Business constraints
+
+[role="arc42help"]
+****
+.Contents
+The business constraints for the development of the system.
+
+.Motivation
+Know the business constraints for the system.
+
+.Form
+Table with all the business constraints and its corresponding description .
+****
+The business constraints are those decisions affecting the software architecture design or that will affect future decisions.
+
+[options="header",cols="1,2"]
+|===
+|Constraint|Description
+| _GitHub_|The code has to be deployed in GitHub, a public repository.
```

Comment:
I feel like Github is more of a technical constraint rather than a business one, because it does not really affect architecture.